addons/payment_asiapay/views/payment_provider_views.xml

```diff
@@ -24,6 +24,19 @@
                            groups="base.group_no_one"/>
                 </group>
             </group>
+
+            <field name="maximum_amount" position="after">
+                <label for="available_currency_ids"/>
+                <!-- Use `o_row` to allow placing a button next to the field in overrides. -->
+                <div name="available_currencies" class="o_row">
+                    <field name="available_currency_ids"
+                        widget="many2many_tags"
+                        placeholder="Select currencies. Leave empty not to restrict any."
+                        context="{'active_test': False}"
+                        options="{'no_create': True}"
+                        required="state != 'disabled'"/>
+                </div>
+            </field>
```

Comment:
Ho, found it!! =D Hopefully, every person needing that will install Asiapay =P
This should stay in `payment`, and add condition for it to be required in Asiapay, but staying falsy without =]